docs/source/tutorials/output_and_config.rst

```diff
@@ -259,10 +259,10 @@ You can also skip rendering all animations belonging to a section like this:
 .. code-block:: python
 
     def construct(self):
-        self.next_section(skip_animations=True)
         # play some animations that shall be skipped...
-        self.next_section()
+        self.next_section(skip_animations=True)
         # play some animations that won't get skipped...
+        self.next_section()
```

Comment:
I agree with the observation in our Discord that this change doesn't really improve the structure, I'd rather see it reverted.

Reply:
Reverted! [`11a586c`](https://github.com/ManimCommunity/manim/pull/3412/commits/11a586c5b3408a5adfc2613b512ed7e8d9399bc1)